webpack/assets/javascripts/react_app/components/componentRegistry.js

```diff
@@ -182,6 +182,8 @@ const coreComponets = [
   { name: 'WelcomeArchitecture', type: WelcomeArchitecture },
 ];
 
-componentRegistry.registerMultiple(coreComponets);
+if (!componentRegistry.registry[coreComponets[0].name]) {
+  componentRegistry.registerMultiple(coreComponets);
+}
```

Comment:
Can we use this opportunity to finally add "n" to `coreComponents`? :innocent:

Reply:
lol sure :)